### DISCUSSION TITLE
doris 数据大小写问题

### DISCUSSION THREAD

**Discussion by liuanxin** [apache/doris#21256] [RFC]
碰到一个从 doris 基于 jdbc_catelog 写入 mysql 数据的问题, mysql 因为表的字符集是 _ci 结尾的, 数据忽略大小写, 但是 doris 中却是大小写敏感的, 这导致当遇到 mysql 中是主键或唯一索引时会报 `Duplicate entry 'xxxx' for key 'PRIMARY'` 的错误

想知道 doris 是否也有类似的字符集, 在处理数据时, 也是能忽略大小写的?

**Comment by yagagagaga** [comment]
Doris 目前字符集只支持utf8，处理数据时，无法忽略大小写。
